wiki/Ranking_Criteria/en.md

```diff
@@ -225,7 +218,7 @@ This category contains explicit allowance statements of concepts and rules that
 - **Acceptable songs with mature lyrics/themes must be marked with an 18+ in the beatmap's description.** This game is for all ages, and so a warning is needed for younger audiences. Some songs, however, are unacceptable for ranking regardless of an 18+ warning. See [song content rules](/wiki/Ranking_Criteria/Song_Content_Rules) for more details.
 - **A beatmapset may only contain one song file used by all difficulties.** Multiple song files within a single beatmap set is unsupported and results in unexpected behaviour with preview times, metadata, etc.
 - **A song's audio file must be of reasonable quality.** Try to find the highest quality source file available rather than ripping a file from a streaming video website. Songs should be normalized to their original release volumes.
-- **A beatmapset's audio file must be no lower than 128kbps and no higher than 192kbps.** Variable bit rate songs must average between that range.
+- **A beatmapset's audio file must be no higher than 192kbps.** Variable bit rate songs must average between that range.
```

Comment:
"between that range" when there's only a top limit now so there is no range

-> Variable bit rate songs must average beneath this amount.